File: src/day2/Kreis.java
package day2;

public class Kreis {

	private double radius;
	
	public Kreis() {
		super();
	}

	public Kreis(double radius) {
		this.radius = radius;
	}
	
	public double getUmfang() {
		return 2 * radius * Math.PI;
	}

	public double getFlaeche() {
		return radius * radius * Math.PI;
	}

	public String toString() {
		return "Kreis mit Radius=" + this.radius + ", Umfang=" + this.getUmfang() + ", Fläche=" + this.getFlaeche(); 
	}
	
	public double getRadius() {
		return radius;
	}

	public void setRadius(double radius) {
		this.radius = radius;
	}

}

File: lab/04-classes/kreis.py
import math

class Kreis:

    def __init__(self, radius):
        self.radius = radius

    def umfang(self):
        return 2 * self.radius * math.pi

    def flaeche(self):
        return math.pi * pow(self.radius, 2)

    def print(self):
        print("Kreis mit Radius: ", self.radius)
        print("- Umfang =", self.umfang())
        print("- Fläche =", self.flaeche())



if __name__ == '__main__':
    radien = [2, 4]
    kreise = []
    for radius in radien:
        kreis = Kreis(radius)
        kreise.append(kreis)
    for kreis in kreise:
        kreis.print()
        kreis.radius
